.github/workflows/deploy-pages.yaml

```diff
@@ -51,7 +49,45 @@ jobs:
 
     # Push the book's HTML to github-pages
     - name: GitHub Pages action
-      uses: peaceiris/actions-gh-pages@v3.6.1
+      uses: peaceiris/actions-gh-pages@v4.0.0
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         publish_dir: ./docs/_build/html
+        # destination_dir: docs
```

Comment:
I think this was added while you were trying a few things, so it could be removed

Reply:
removed